notebooks/_06-Architecture.md

```diff
@@ -154,7 +165,7 @@ Causal graphs will be output to .png files and pushed to the repo
 
 Summaries of model estimate will be saved as markdown tables
 
-Plots of results will be saved as .png files and pushed to the repo
+Plots of results will be saved as .png files and pushed to the repo and included in the final presentation
 
 Previously estimated models will be stored in a dictionary that can be pickled and loaded later.
 ### 7.6 How will created data be validated?
```

Comment:
Below, I imagine it should be "Ensure all columns have the correct types" as opposed to `tyoes`.

Reply:
yes. will change.